assignment/scripts/describe.js

```diff
@@ -45,9 +45,9 @@ if (name === 'Mary') {
 */
 
 // 2. WRITE YOUR DESCRIPTION HERE
-// 
-// 
-// 
+// The secret variable is created, which may be filled in later
+// Assign variable code = 123, as code=123, secret = 'super', code *2, code is now = 246
```

Comment:
This line is very difficult to read. I see you are trying to squeeze a lot of information into one line, but the combination of English and JS syntax is harder to read than either would be on their own. What about something like:

```suggestion
// Create a variable 'code' with the value 123
// Check if code is equal to 123, it is so set secret to 'super' and multiply code by two
```